preprocessors/openstreetmap/osm_service.py

```diff
@@ -67,6 +67,7 @@ def server_config2(url, bbox_coord):
     (node({lat_min},{lon_min},{lat_max},{lon_max}) ["amenity"];
     way({lat_min},{lon_min},{lat_max},{lon_max}) ["amenity"];
     rel({lat_min},{lon_min},{lat_max},{lon_max}) ["amenity"];
+    way({lat_min},{lon_min},{lat_max},{lon_max}) ["building"];
```

Comment:
The ```building``` key contains almost the same information on the points of interest as the ```amenity``` key, but none of the two has complete data, hence the inclusion in case any information is available in one but missing in the other.